common/trinary/flex_trit.h

```diff
@@ -43,24 +43,28 @@ typedef int8_t flex_trit_t;
 #define FLEX_TRIT_SIZE_243 243
 #define FLEX_TRIT_SIZE_6561 6561
 #define FLEX_TRIT_SIZE_8019 8019
+#define NUM_TRITS_FOR_FLEX_TRIT 1
```

Comment:
@ifullgaz is adding them too in his #346 maybe you'll have to coordinate on this

Reply:
I noticed